packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -139,6 +140,8 @@ class PinCodeBloc extends $PinCodeBloc {
         _pinCode.value = '';
         throw ErrorWrongPin(errorMessage: 'Wrong Pin');
       }
+    } else if (digits == 0) {
+      return true;
```

Comment:
I think this is not correct. Shouldn't we show the button if there's a stored PIN and biometrics are enabled?